bpf/lib/encap.h

```diff
@@ -137,14 +137,22 @@ encap_and_redirect(struct __sk_buff *skb, struct endpoint_key *k,
 	}
 
 #ifdef ENABLE_IPSEC
-	if (from_host)
-		return enacap_and_redirect_nomark_ipsec(skb, tunnel->ip4,
-						        seclabel, monitor);
-	else
-		return encap_and_redirect_ipsec(skb, tunnel->ip4, seclabel, monitor);
-#else
-	return __encap_and_redirect_with_nodeid(skb, tunnel->ip4, seclabel, monitor);
+	if (tunnel->key) {
+		if (from_host)
+			return enacap_and_redirect_nomark_ipsec(skb,
```

Comment:
Nit, typo: `enacap` ?

Reply:
hmm and I must have cut'n'pasted it throughout? ... Will fix because I can't think up what I meant by enacap at the moment.

Reply:
I'll do this in another rename patch seeing its started earlier in the tree.